dist/rpm/os-autoinst.spec

```diff
@@ -107,15 +105,11 @@ for i in 07-commands 13-osutils 14-isotovideo 18-qemu-options 18-backend-qemu 99
 done
 
 %build
-mkdir -p m4
-autoreconf -f -i
-%configure --docdir=%{_docdir}/%{name}
-make INSTALLDIRS=vendor %{?_smp_mflags}
+%cmake -DOS_AUTOINST_DOC_DIR:STRING=%{_docdir}/%{name}
```

Comment:
Why is the special dir variable needed?

Reply:
I don't know what `%{_docdir}` might possibly resolve to. In our builds for x86_64 TW it would resolve to the same what the build system would use by default anyways. However, this was explicitly specified before and I saw no reason to change that considering I don't know all possible values of `%{_docdir}` and I don't want to break things. (We can still improve this later.)